ui/templates/base_error.html

```diff
@@ -22,11 +22,12 @@
 	  		{% if authenticated %}
 	  		<div class="dropdown btn-group">
               <a class="btn btn-danger" href="/dashboard"><span>{{ name }}</span></a>
```

Comment:
this file existed before the PR. it seems that it is here to serve an error page without running any react code. i'm not 100% certain why it's necessary. @aliceriot or @noisecapella might be able to explain it